src/sentry/static/sentry/app/views/admin/adminLayout.jsx

```diff
@@ -1,67 +1,67 @@
 import DocumentTitle from 'react-document-title';
 import React from 'react';
+import styled from '@emotion/styled';
 
+import space from 'app/styles/space';
 import Hook from 'app/components/hook';
 import ListLink from 'app/components/links/listLink';
-import Sidebar from 'app/components/sidebar';
```

Comment:
Why get rid of sidebar?

Reply:
It's rendered by the parent component. This actually did nothing but render twice. It being there was just wrong.

Reply:
This is good info for the PR description

Reply:
yeah good point :)